driver.py

```diff
@@ -426,17 +428,17 @@ def maybe_create_directory_on_remote(self, realpath):
 
     def sync_completed_files_to_remote(self, realpath, completed_files):
         tmpfile_path = None
-        
+
         with tempfile.NamedTemporaryFile(
-            suffix=".lst", prefix="transfer_list-", delete=False
+                suffix=".lst", prefix="transfer_list-", delete=False
         ) as transfer_list:
             tmpfile_path = transfer_list.name
```

Comment:
Same applies here, although this is not a definition so I'm not sure if the 8-spaces is correct according to PEP8?

Reply:
As in the previous comment, agreed.

Reply:
Resolved in aab4917.